cmd/finch/nerdctl.go

```diff
@@ -121,10 +122,27 @@ func (nc *nerdctlCommand) run(cmdName string, args []string) error {
 				arg = fmt.Sprintf("%s%s", arg[0:11], resolvedIP)
 			}
 			nerdctlArgs = append(nerdctlArgs, arg)
+		case arg == "--sign=cosign" || arg == "--verify=cosign":
```

Comment:
We would need to also add `"--sign cosign"` and `"--verify cosign"` if we go with the current approach. +1 on https://github.com/runfinch/finch/pull/296/files#r1142351036